app/views/softdrinksindustrylevy/register/smallProducerConfirm.scala.html

```diff
@@ -0,0 +1,44 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.play.views.html.helpers._
+@import sdil.config.AppConfig
+
+@(backLink: Call)(implicit request: Request[_], messages: Messages, config: AppConfig)
+
+@views.html.main_template(Messages("sdil.common.title")) {
+    <a class="link-back" href="@backLink.url">@Messages("sdil.common.back")</a>
+
+    <legend>
+        <h1 class="heading-large">@Messages("sdil.small-prod-confirm.heading")</h1>
+    </legend>
+    <p class="lede">
+        @Messages("sdil.small-prod-confirm.lede")
+
+    </p>
+    <p></p>
```

Comment:
Have tested this in the prototype and we don't actually need the empty `<p>` so this can be removed